### ISSUE TITLE
Default exports and no-color-literals

### ISSUE THREAD

**Issue by pr0da** [Intellicode/eslint-plugin-react-native#195]
Hi!
The `no-color-literals` does not mark the following case as error:

```js
export default StyleSheet.create({
  container: {
    color: '#fff'
  }
});
```

If I pull out the stylesheet into a variable and export that, than it works as expected.
```js
const styles = StyleSheet.create({
  container: {
    color: '#fff'
  }
});
export default styles;
```

The first version should omit an error as well.
